src/QlrBrowser/core/filesystemmodel.py

```diff
@@ -65,7 +65,7 @@ def filtered(self, filter):
         if self.isdir:
             if namematch:
                 # Stop searching. Return this dir and all sub items
-                return FileSystemItem(self.fullpath, True)
+                return FileSystemItem(self.fullpath, False)
```

Comment:
@bsnizek This means that when folder `A` is a search hit, the files in and below `A` will not be seen in the browser. This is not very user friendly. When a directory is a search hit - everythin in and below is interesting to the user.